pipeline/dags/dora_rgpd_notifications.py

```diff
@@ -0,0 +1,78 @@
+import logging
+
+import airflow
+from airflow.operators import empty, python
+import pendulum
+
+from dag_utils.virtualenvs import PYTHON_BIN_PATH
+
+default_args = {}
+
+logger = logging.getLogger(__name__)
+
+
+
+def _sync_new_contacts_to_brevo():
+    from airflow.models import Variable
+
+    from dag_utils import constants
+    from data_inclusion.scripts.tasks import brevo
+    from dags.dag_utils import pg
+
+    all_emails = [
+        row[0]
+        for row in pg.hook().get_records(
+            sql="SELECT courriel FROM dora.int_dora__contacts",
+        )
+    ]
+
+    brevo_contacts = [
+        row[0]
+        for row in pg.hook().get_records(
+            sql="SELECT * FROM brevo.all_rgpd_dora_contacts",
```

Comment:
```suggestion
            sql="SELECT courriel FROM brevo.all_rgpd_dora_contacts",
```
?

Reply:
`email` plutot. On a ça:
```
         email         | id  | emailBlacklisted | smsBlacklisted |           createdAt           |          modifiedAt           
```

Reply:
Ca me donne envie d'ecrire un petit test end to end.

Reply:
tu vois comment on pourrait s'y prendre ?